msw/2/type-args/index.ts

```diff
@@ -128,9 +128,27 @@ function getCallbackData(
 	return [callbackBody, cbParams, syntaxCb];
 }
 
-const isNeitherNullNorUndefined = <T>(
+function isNeitherNullNorUndefined<T>(
 	t: NonNullable<T> | null | undefined,
-): t is NonNullable<T> => t !== null && t !== undefined;
+): t is NonNullable<T> {
+	return t !== null && t !== undefined;
+}
+
+function deleteByPos(
+	sourceFile: SourceFile,
+	positionObject: Record<number, string>,
+) {
+	let offset = 0;
+	Object.entries(positionObject)
+		.sort(([pos1], [pos2]) => +pos1 - +pos2)
```

Comment:
hmm why `+` in front of it? are you casting `pos1` and `pos2`?

Reply:
typescript casts keys to string when using `Object.` methods, i think that's why
in reality, no cast is happening @grzpab